BLSQ/iaso
# IA-947: save precision and altitude in submission details + add info in csv/xlsx export

- Missing data needed in Instance location details (altitude and accuracy)
- Those infos needed to be also in the csv & xlsx exports

Related JIRA tickets : [IA-947](https://bluesquare.atlassian.net/browse/IA-947?atlOrigin=eyJpIjoiM2NhY2FlNDQ5NWZlNDZkMTg1OTcwMDNmODk0NjhhODMiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- add accuracy to Instance Model
- refactor functional component `InstanceDetailsLocation` with Typescript
- update instance type
- update dictionary to add accuracy for the export

## How to test

### Show location info

- Make sure you can link a submission to an orgunit ([How to link an instance to an org unit](https://wiki.bluesquare.org/en/Dev-team/product-management/iaso/show_link_instance_reference))
-  Go to submission list, choose a submission linked to an org unit
-  Go to Map tab in Orgunit Detail and add a location marke then go back to the submission detail and see if those infos are displayed in location details
- Go to Django Admin and choose a submission, add a marker on the map and fill the accuracy field -> then go back to instance view : you should see those coordinates now instead of the values from the org unit

### Export file 

- filter submission by form
- select the form you used to test this PR
- export csv/xlsx
- Make sure you see the correct values in each column

## Print screen / video

### Show location info


https://user-images.githubusercontent.com/25134301/234865707-f474c4fe-1a59-4ace-807e-eb3609ca2a5a.mov



### Export file

https://user-images.githubusercontent.com/25134301/234842207-527cb700-01a0-4fdd-ba54-8977e56fe08e.mov

[IA-947]: https://bluesquare.atlassian.net/browse/IA-947?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
